### PR TITLE
CI: stylistic changes to simplify script

### PR DESCRIPTION
**Pull request by compnerd**
Use CMake more effectively to both build and change the working
directory.  This simplifies the number of steps and reduces possible
error points.

Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

Resolves SR-NNNN.

### CODE REVIEW

**Comment by compnerd** [comment]
@swift-ci please clean test windows platform

**Comment by compnerd** [comment]
CC: @drodriguez

File: utils/build-windows.bat [hunk 1]
```diff
@@ -272,10 +263,7 @@ endlocal
 :: Configures, builds, and installs LLDB
 setlocal enableextensions enabledelayedexpansion
 
-mkdir "%build_root%\lldb" %exitOnError%
-pushd "%build_root%\lldb"
-
-cmake "%source_root%\lldb"^
+cmake^
```
Comment (drodriguez) [inline utils/build-windows.bat]: Missing `-B`?

**Review by drodriguez — CHANGES_REQUESTED** [review]
I think LLDB is missing the build directory, otherwise looks good.

File: utils/build-windows.bat (hunk 1 above)
Comment (compnerd) [reply utils/build-windows.bat]: mechanical changes are difficult!

**Comment by compnerd** [comment]
@swift-ci please clean test Windows platform

**Comment by compnerd** [comment]
@swift-ci please clean test Windows platform

**Comment by compnerd** [comment]
@swift-ci please clean test Windows platform

**Review by drodriguez — APPROVED** [review]

**Comment by compnerd** [comment]
@swift-ci please clean test Windows platform

**Comment by compnerd** [comment]
@swift-ci please smoke test

**Pull request merged**